src/audio/src/src.c

```diff
@@ -193,7 +193,7 @@ int src_buffer_lengths(struct src_param *a, int fs_in, int fs_out, int nch,
 		 * variable number of blocks to process per each stage
 		 * there is no equation known for minimum size.
 		 */
-		a->sbuf_length = 2 * nch * stage1->blk_out * r1;
+		a->sbuf_length = 2 * nch * stage1->blk_out * r1 + 8;
```

Comment:
Can we add the ```FIXME:``` in the comment too so it's easy to follow.

Reply:
and why do we always add 8 ?

Reply:
That was the hack, I think the right way is to experimentally find the multiplier like 2.05 and do here fractional multiply.

Reply:
This is now fixed with SBUF_LENGTH_MULTIPLIER set to fractional value 2.4. All tests passed without freeze with value 2.3, so I chose to use 2.4 here. I don't have a closed form equation for exact buffer size need unfortunately.